vck/src/commonMain/kotlin/at/asitplus/wallet/lib/agent/IssuerAgent.kt

```diff
@@ -35,6 +50,7 @@ import kotlin.time.Duration.Companion.hours
 /**
  * An agent that only implements [Issuer], i.e. it issues credentials for other agents.
  */
+
 class IssuerAgent(
```

Comment:
Parameter `revocationListLifetime` is now never used. Shouldn't that be used in `buildStatusListTokenPayload()`?

Reply:
fixed in 93ce860696aa4077acb9a053a8b3eac9bee2733b